src/components/BulkEnrollmentPage/stepper/BulkEnrollmentStepper.jsx

```diff
@@ -96,7 +67,7 @@ const BulkEnrollmentStepper = ({ subscription, enterpriseSlug, enterpriseId }) =
               enterpriseId={enterpriseId}
               enterpriseSlug={enterpriseSlug}
               subscription={subscription}
-              returnToInitialStep={() => setCurrentStep(ADD_COURSES_STEP)}
+              returnToInitialStep={() => { setCurrentStep(ADD_COURSES_STEP); onEnrollComplete(); }}
```

Comment:
nit: I wonder if we should rename returnToInitialStep to onEnrollComplete/something similar here.

Reply:
yeah I am of the same opinion too, the concept of returning to initial step actually no longer exists either

Reply:
agreed, the returning to initial step actually doesn't make sense anymore, it's more like do something once enrollment ends